gccontrol/sampler.go

```diff
@@ -14,18 +14,20 @@ const (
 )
 
 type sampler struct {
-	lastFinished int64
-	curr         int64
-	count        int
-	past         []int64
+	next         int     // Next index in the past slice.
+	past         []int64 // History of sample size between collections.
+	lastFinished int64   // Last number of finished requests.
+	curr         int64   // Current sample size.
```

Comment:
I believe that "current sample size" could be misunderstood as "the size of sampler" since newSampler receives a size as parameter, but that's just a detail.

Reply:
It is important. Good catch! Fixed.